### ISSUE TITLE
Can LeafMachine detect field photos on herbarium sheets

### ISSUE THREAD

**Issue by nickynicolson** [Gene-Weaver/LeafMachine2#18]
I have noticed that some specimens from the Flora of Zambia project include a field photo - e.g. Kew specimen [K001614477](https://www.gbif.org/tools/zoom/simple.html?src=//api.gbif.org/v1/image/cache/occurrence/4519448288/media/9b87d574bd20d81a2b122bc46e453975) - can leaf machine detect these?

**Comment by Gene-Weaver** [comment]
Yes! I have never done an exhaustive test of the photo detection ability, but LM2 is definitely trained to detect photos in images, along with maps, envelopes, and attached items. I processed the image you shared and this is what we end up with:

Archival component detector:
![kew_archival](https://github.com/user-attachments/assets/c681b5a9-0760-46c4-b1c5-94d3e9402835)

Plant component detector:
![kew_plant](https://github.com/user-attachments/assets/098aaba1-fcd4-48fb-aefa-ed4a5b475ca1)

Summary Image:
![kew_summary](https://github.com/user-attachments/assets/d92b0b62-4b3e-4612-ad21-cdea992dfa04)
